app/app/test/page.jsx

```diff
@@ -1,87 +1,92 @@
+/* eslint-disable @next/next/no-img-element */
 'use client';
 
-import React, { useState, useEffect } from 'react';
-import QuestionCard from '../../components/questioncard';
-import PreviousNext from '../../components/prevNext';
+import { useRouter } from 'next/navigation';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faArrowsRotate,
+  faClock,
+  faForward,
+  faListCheck,
+} from '@fortawesome/free-solid-svg-icons';
+import Button from '@/components/Button';
 
-function exam() {
-  const [questions, setQuestions] = useState([]);
-  const [currentQuestionIndex, setCurrentQuestionIndex] = useState(0);
-  const [userAnswers, setUserAnswers] = useState({});
+const ExamInstructions = () => {
+  const router = useRouter();
 
-  useEffect(() => {
-    const fetchQuestions = async () => {
-      try {
-        const response = await fetch('http://localhost:5002/questions');
-        if (!response.ok) {
-          throw new Error('Failed to fetch questions');
-        }
-        const data = await response.json();
-
-        const shuffledQuestions = data
-          .sort(() => Math.random() - 0.5)
-          .slice(0, 40);
-        setQuestions(shuffledQuestions);
-      } catch (error) {
-        console.error('Error fetching questions:', error);
-      }
-    };
-
-    fetchQuestions();
-  }, []);
-
-  const handleAnswer = (answer) => {
-    const updatedAnswers = { ...userAnswers, [currentQuestionIndex]: answer };
-    setUserAnswers(updatedAnswers);
-    if (currentQuestionIndex < questions.length - 1) {
-      setCurrentQuestionIndex(currentQuestionIndex + 1);
-    }
-  };
-
-  const handleNext = () => {
-    if (currentQuestionIndex < questions.length - 1) {
-      setCurrentQuestionIndex(currentQuestionIndex + 1);
-    }
-  };
-
-  const handlePrevious = () => {
-    if (currentQuestionIndex > 0) {
-      setCurrentQuestionIndex(currentQuestionIndex - 1);
-    }
+  const handleBeginExam = () => {
+    router.push('/questions');
```

Comment:
Since you doing this on click, the better way would be to use Link component instead. If you use Link, Next will prefetch some of the data and next page will load faster